### DISCUSSION TITLE
问问大家，我可以玩崩铁吗

### DISCUSSION THREAD

**Discussion by xiangxliuying** [share121/inter-knot#551] [其他]
如题
![Screenshot_2024-08-08-15-12-30-26_589d4bb4f12f057e26f1c0af39598661](https://github.com/user-attachments/assets/6425573a-764f-4294-9f50-c011ad548137)
![Screenshot_2024-08-08-15-12-24-90_589d4bb4f12f057e26f1c0af39598661](https://github.com/user-attachments/assets/c5bd818b-4a28-45e5-9aae-86ba2b448c94)

**Comment by Ar10o01** [comment]
你不可以！！！

**Comment by ZhanPai123** [comment]
你不允许玩星穹铁道

**Comment by FrostPhoenix79** [comment]
![Screenshot_20240730_235747](https://github.com/user-attachments/assets/5cf49b4e-4e81-498c-935a-a027b521cefa)
你能玩，我就能玩（狗头保命）

**Reply by StarrySkyehk** [reply]
啊？

**Reply by FrostPhoenix79** [reply]
（肯定）

**Comment by StarrySkyehk** [comment]
啊？不是

**Comment by npldqt** [comment]
beyond的不许玩

**Comment by Bilistar333** [comment]
啊。。。

**Comment by Qijoten** [comment]
这号废了，给我吧

**Comment by Mooshed88-a** [comment]
我看着我全歪的遗器陷入沉思

**Comment by lizard315315** [comment]
凭什么！
